apps/frontend/src/shared-research/__tests__/api.test.ts

```diff
@@ -105,6 +105,31 @@ describe('getResearchOutputs', () => {
     );
   });
 
+  it('uses teamId as a filter', async () => {
+    await getResearchOutputs(mockIndex, {
+      ...options,
+      teamId: '12345',
+    });
+
+    expect(mockIndex.search).toHaveBeenLastCalledWith(
+      '',
+      expect.objectContaining({ filters: 'team.id:"12345"' }),
+    );
+  });
+
+  it('adds teamId to other filters', async () => {
```

Comment:
you may wanna add a test that uses a couple of type filters and a team filter. I would expect it to produce a query with brackets around the types which I dont think it will

Reply:
Good catch 👍